### PR TITLE
margin for menu is fixed

### PR DESCRIPTION
**Pull request by niharikachavati**

### CODE REVIEW

File: src/Components/Dashboard/MenuList.css
Comment (saiviru) [inline src/Components/Dashboard/MenuList.css:1]: Why this change in css file when the same is done in js file?

Comment (niharikachavati) [reply src/Components/Dashboard/MenuList.css:1]: i have done this change  in jsx file only

**Review by saiviru — APPROVED** [review]
